### PR TITLE
Initial code drop

### PR DESCRIPTION
**Pull request by brianjmurrell**
Fix spelling error in README.md.

Add a spec file due to pyp2rpm deficiencies.
- override pyp2rpm's Requires: python2-urllib3 >= 1.8 as
  urllib3 has not been migrated to python{2,3} yet

Add .travis.yml.

Add badges.

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # python-requests-unixsocket
-Packating/buiding for python-requests-unixsocket
+Packaging/buiding for python-requests-unixsocket
+
+
+[![Build Status](https://travis-ci.org/intel-hpdd/python-requests-unixsocket.svg?branch=master)](https://travis-ci.org/intel-hpdd/python-requests-unixsocket)
+
+[![Build Status](https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/package/python-requests-unixsocket/status_image/last_build.png)](https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/package/python-requests-unixsocket/)
```
Comment (jgrund) [inline README.md]: cool.

File: include/hooks/commit-msg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+SOB=$(git var GIT_AUTHOR_IDENT | sed -n 's/^\(.*>\).*$/Signed-off-by: \1/p')
```
Comment (johnsonw) [inline include/hooks/commit-msg]: Does this automatically add in the identity to the commit message?

File: include/travis/virus-scan.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+sudo apt-get install clamav -y
+sudo freshclam
+clamscan --quiet -r ./
```
Comment (johnsonw) [inline include/travis/virus-scan.sh]: nice 👍

File: include/hooks/commit-msg (hunk 2 above)
Comment (brianjmurrell) [reply include/hooks/commit-msg]: It adds the "Signed-off-by:" yes.  You have to run `make githooks` in your local clone (that as a _module-tools_ subrepo) to get the hooks set up though.

**Pull request merged**